File: DAY_1/TwoSum.java
package DAY_1;
    import java.util.*;

        public class TwoSum {
            private int[] array;
            private int target;
            private List<int[]> result;

            public TwoSum() {
                this.result = new ArrayList<>();
            }

            public void getInput(Scanner sc) {
                String[] arrStr = sc.nextLine().split(",");
                array = new int[arrStr.length];
                for (int i = 0; i < arrStr.length; i++) {
                    array[i] = Integer.parseInt(arrStr[i].trim());
                }
                target = Integer.parseInt(sc.nextLine().trim());
            }

            public List<int[]> twoSum() {
                result.clear();
                int n = array.length;
                for (int i = 0; i < n; i++) {
                    for (int j = i + 1; j < n; j++) {
                        if (array[i] + array[j] == target) {
                            result.add(new int[]{i, j});
                            break;
                        }
                    }
                }
                if (result.isEmpty()) {
                    return null;
                }
                return result;
            }

            public void display() {
                if (result != null && !result.isEmpty()) {
                    int[] pair = result.get(0);
                    System.out.println("(" + pair[0] + ", " + pair[1] + ")");
                } else {
                    System.out.println(-1);
                }
            }

            public static void main(String[] args) {
                Scanner sc = new Scanner(System.in);
                TwoSum ts = new TwoSum();
                ts.getInput(sc);
                ts.twoSum();
                ts.display();
                sc.close();
            }
        }

File: DAY_1/Two_sum.py
class Solution():
    def __init__(self):
        self.array = self.target = None
        self.r = []

    def get_input(self):
        self.array = list(map(int, input().split(",")))
        self.target = int(input())
    
    def two_sum(self):
        self.l = len(self.array)
        self.r = []
        for i in range(self.l):
            for j in range(i+1, self.l):
                if self.array[i] + self.array[j] == self.target:
                    self.r.append((i, j))
                    break
        if not self.r:
            return -1
        return self.r

    def display(self):
        print(self.r[0])



if __name__ == "__main__":
    S = Solution()
    S.get_input()
    S.two_sum()
    S.display()